.codecov.yml

```diff
@@ -0,0 +1,12 @@
+---
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 1
+        base: auto
+comment: false
+ignore:
+  - "autoload/go/*_test.vim"
+  - "autoload/gotest.vim"
```

Comment:
Probably want to ignore everything that doesn't match `*.vim` (or `*.vim~*_test.vim`)? Looks like you can use regexps: https://docs.codecov.io/docs/ignoring-paths

Reply:
Actually, I think we can ignore everything outside `autoload/go`? We have one file outside of that right now (`test/gopath_test.vim`), but that can/should probably be moved to `autoload/go/path_test.vim`.